README.md

```diff
@@ -146,7 +146,7 @@ Usage of ./ffuf:
   -mode string
     	Multi-wordlist operation mode. Available modes: clusterbomb, pitchfork (default "clusterbomb")
   -mr string
-    	Match regexp
+    	Match regexp, also fuzz keyword allowed
```

Comment:
I don't think this needs a mention in the help text, as I think the default assumption here is that everything works with the keywords ;)

Reply:
Removed.